src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -76,7 +76,6 @@
                   class="kuiLink"
                   href="https://en.wikipedia.org/wiki/ISO_week_date"
                   target="_blank"
-                  title="Wikipedia: ISO Week Date"
```

Comment:
We should never use `title`, and this element contains text so it's already accessible.